packages/compartment-mapper/src/import-hook.js

```diff
@@ -45,13 +45,27 @@ function getImportsFromRecord(record) {
   return (has(record, 'record') ? record.record.imports : record.imports) || [];
 }
 
+// Node.js resolution allows for an incomplement specifier.
+// https://nodejs.org/api/modules.html#all-together
+const nodejsConventionCandidateSuffixes = [
+  // LOAD_AS_FILE(X)
+  '.js',
+  '.json',
+  '.node',
+  // LOAD_INDEX(X)
+  '/index.js',
+  '/index.json',
+  '/index.node',
+];
+
 /**
  * @param {ReadFn|ReadPowers} readPowers
  * @param {string} baseLocation
  * @param {Sources} sources
  * @param {Record<string, CompartmentDescriptor>} compartmentDescriptors
  * @param {Record<string, any>} exitModules
  * @param {HashFn=} computeSha512
+ * @param {Array<string>} candidateSuffixes
```

Comment:
“Pass [] to emulate Node.js’s strict behavior. The default handles Node.js’s CommonJS behavior. Unlike Node.js, the Compartment Mapper lifts CommonJS up, more like a bundler, and does not attempt to vary the behavior of resolution depending on the language of the importing module.”